packages/account-v2/src/pages/CurrencySelector/CurrencySelector.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { useState } from 'react';
+import Actions from '../../components/SignupWizard/Actions';
+import WizardScreenWrapper from '../../components/SignupWizard/WizardScreenWrapper';
+import Currencies from './Currencies';
+import { CURRENCY_TYPES } from './helpers';
+
+const CurrencySelector = () => {
+    const [currency, setCurrency] = useState('');
+
+    // TODO: Remove this console.log
+    // eslint-disable-next-line no-console
+    console.log(currency);
+    return (
+        <WizardScreenWrapper>
+            <div className='w-full pt-2400'>
```

Comment:
are we handling allowed currencies logic in a different PR?

Reply:
Yes @amina-deriv I'll create another PR for the logic to handle the logic for available currencies to keep things small and simpler